src/content/release/breaking-changes/android-surface-plugins.md

```diff
@@ -0,0 +1,94 @@
+---
+title: New APIs for Android plugins that render to a Surface
+description: >
+  Adds a new API, `SurfaceProducer`, to the Android embedding API, which
+  transparently handles the creation and management of a `Surface` for plugins.
+  For Impeller, use of this API is required.
+---
+
+## Summary
+
+The Android embedder for Flutter introduces a new API, [`SurfaceProducer`][],
+which allows plugins to render to a `Surface` without needing to manage what
+the backing implementation is. Plugins using the older
+[`createSurfaceTexture`][] API will not work with [Impeller][].
+
+## Background
+
+An Android [`SurfaceTexture`][] is a backing implementation for a [`Surface`][]
+that uses an [OpenGLES][] texture as the backing store.
+
+For example, a plugin might display frames from a _camera_ plugin:
+
+![Flowchart](https://camo.githubusercontent.com/cdb52c5d371b4f1d5573b650a0eddb0871e5e8be1012d290e008f41bc71b2580/68747470733a2f2f736f757263652e616e64726f69642e636f6d2f7374617469632f646f63732f636f72652f67726170686963732f696d616765732f636f6e74696e756f75735f636170747572655f61637469766974792e706e67)
+
+In newer versions of the Android API (>= 29), Android introduced a
+backend-agnostic [`HardwareBuffer`][], which coincides with the minimum version
+that Flutter will attempt to use the [Vulkan][] renderer. In order to support
+this, the Android embedding API needed to be updated to support a more generic
+`Surface` creation API that doesn't rely on OpenGLES.
+
+## Migration guide
+
+If you are using the older [`createSurfaceTexture`][] API, you should migrate to
+the new [`createSurfacePRoducer`][] API. The new API is more flexible and allows
+the Flutter engine to transparently pick the best implementation for the
```

Comment:
Here and elsewhere I would drop transparently, since actually its opaque/hidden!

Reply:
Done!